modules/face/samples/sampleKazemi.cpp

```diff
@@ -0,0 +1,137 @@
+/*
```

Comment:
Again remark header.
This sample is actually training the model right? I would add this in the filename. Now it seems this is another function next to the detectLandmarks sample.